engine/runtime-integration-tests/src/test/scala/org/enso/polyglot/test/ApiTest.scala

```diff
@@ -1,14 +1,17 @@
-package org.enso.polyglot
+package org.enso.polyglot.test
 
 import org.enso.common.RuntimeOptions
+import org.enso.polyglot.PolyglotContext
 import org.graalvm.polyglot.Context
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 
+import java.io.ByteArrayOutputStream
 import java.nio.file.Paths
 
 class ApiTest extends AnyFlatSpec with Matchers {
   import org.enso.common.LanguageInfo._
+  val out = new ByteArrayOutputStream()
```

Comment:
Do we do anything with it? We certainly don't close it.

Reply:
The whole `executionContext` is not closed in these tests. What's the point of closing `ByteArrayOutputStream`? It will get garbage collected sooner or later. And we don't do anything with it on purpose, this is equivalent to setting `/dev/null` as the output.